shared/src/commonMain/kotlin/com/bumble/livemosaic/model/Entries.kt

```diff
@@ -248,6 +248,12 @@ val entries = listOf(
         githubUserName = "victor-wallapop",
         message = "Why so serious?"
     ),
+    Entry.Image(
+        mosaic = MOSAIC1,
+        githubUserName = ”jeremyleenz",
+        path = "jeremy.jpg",
+        contentDescription = "Enjoying my time at Droidcon London!"
+    ),
```

Comment:
Submission is fine, but CI is failing. Could you please rebase your branch with latest changes in order to have a clean build again? Thanks so much and sorry for the inconveniences 😉

Reply:
Should be okay now hopefully, the problem was mixing `”` with `"`

Reply:
Coool! Approved! See you in the booth for giving your raffle entry.